wiremock/src/main/java/com/cognifide/aem/stubs/wiremock/WireMockStubs.java

```diff
@@ -189,9 +178,21 @@ public WireMockServlet getServlet() {
 
     @AttributeDefinition(
       name = "Global Template Transformer",
-      description = "Enables Pebble template engine / templating"
+      description = "Enables template engine / templating. Handlebars and Pebbles engines are supported"
         + " for response body content and file paths when loading body files. Effectively enables dynamic file loading"
         + " instead of preloading and simplifies defining stubs.")
-    boolean globalTransformer() default true;
+    TransformerEngine globalTransformer() default TransformerEngine.HANDLEBARS;
+
+    @AttributeDefinition(
+      name = "Request journal",
+      description = "Enable the request journal, which records incoming requests for later verification. Protects against reserving too much memory."
+    )
+    boolean requestJournalEnabled() default false;
+
+    @AttributeDefinition(
+      name = "Max Request Journal Entries",
+      description = "Set maximum number of entries in request journal (if enabled). When this limit is reached oldest entries will be discarded. 0 means no limits."
+    )
+    int requestJournalMaxSize() default 0;
```

Comment:
enabled = false is good default for safety reasons. however maxSize = 0 is not safe default when enabled = true. So I recommend to propose some reasonable value > 0 here.

Reply:
set 200 as default